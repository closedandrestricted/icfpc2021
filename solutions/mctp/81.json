{"vertices":[[55,56],[60,48],[58,49],[53,45],[63,30],[54,33],[78,22],[72,25],[63,25],[49,17],[60,23],[13,25],[40,15],[11,27],[48,19],[83,53],[126,26],[114,14],[23,45],[13,65],[56,76],[59,52],[55,26],[65,42],[57,75],[59,14],[60,34],[43,17],[54,35],[34,65],[53,57],[53,48],[61,77],[68,57],[49,51],[62,79],[52,40],[89,41],[76,47],[47,59],[70,9],[58,48],[53,58],[51,31],[77,10],[51,52],[54,41],[57,24],[43,51]]}