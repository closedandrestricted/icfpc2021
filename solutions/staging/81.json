{"vertices":[[58,45],[65,40],[54,39],[58,44],[61,51],[58,60],[16,30],[67,43],[60,48],[34,27],[30,39],[61,88],[32,36],[3,19],[37,29],[95,47],[116,19],[101,10],[80,30],[48,50],[83,33],[70,54],[129,26],[52,47],[80,66],[54,68],[78,66],[75,17],[98,65],[88,67],[81,53],[47,28],[81,15],[69,25],[83,46],[65,23],[63,53],[32,66],[59,57],[85,38],[20,61],[71,47],[79,40],[72,66],[60,75],[80,34],[69,39],[67,59],[62,30]]}