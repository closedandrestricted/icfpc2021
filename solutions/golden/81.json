{"vertices":[[23,30],[14,28],[17,34],[22,30],[51,38],[52,47],[64,44],[61,36],[64,44],[95,47],[85,54],[61,48],[87,42],[70,21],[95,46],[91,65],[108,0],[99,60],[122,22],[100,66],[33,66],[43,30],[61,92],[55,75],[38,30],[77,19],[42,33],[4,18],[57,34],[51,35],[37,44],[78,69],[58,45],[43,49],[30,41],[12,68],[21,38],[81,7],[49,29],[22,45],[85,31],[12,33],[23,39],[45,26],[70,25],[26,44],[20,33],[52,35],[35,12]]}